.gitignore

```diff
@@ -161,3 +161,6 @@ cython_debug/
 
 bin
 local_sandbox
+litellm_uuid.txt
+
+
```

Comment:
- delete the extra lines, only keep 1
- can we have an explanation why this file is added here?

```suggestion
# Reason why this file is added
litellm_uuid.txt
```